src/settings/therapyScreen.js

```diff
@@ -164,40 +164,47 @@ module.exports = class TherapyScreen extends base.Screen {
   }
   async OpenInsulinSensitivitiesScreen() {
     await this.SwipeUpUntilVisible(this.InsulinSensitivitiesLabel);
-    await this.InsulinSensitivitiesLabel.tap();
+    // tapping coordinates by-passes visibility check
+    await this.InsulinSensitivitiesLabel.tap({"x":20,"y":20});
```

Comment:
[nit] How will this work if accessibility text size is changed (thus moving around all of the elements)?

Reply:
Or is this relative to the label so most times it would still be in the label?

Reply:
It is the inner coordinates of the label. typical tap size is 40x40, so this should be good.